library/src/main/java/com/datatorrent/lib/io/fs/AbstractFileOutputOperator.java

```diff
@@ -484,34 +484,40 @@ public FSFilterStreamContext load(@Nonnull String filename)
         //of this operator.
         for(String seenFileName: endOffsets.keySet()) {
           try {
-            Integer part = openPart.get(seenFileName).getValue() + 1;
+            Integer fileOpenPart = this.openPart.get(seenFileName).getValue();
+            int nextPart = fileOpenPart + 1;
```

Comment:
Regd2:
If there is a failure then the file will be corrupted so it will go into that block and recover it. Are we facing any lease issue?